hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/types/UnhealthyContainersResponse.java

```diff
@@ -55,6 +55,23 @@ public class UnhealthyContainersResponse {
   @JsonProperty("replicaMismatchCount")
   private long replicaMismatchCount = 0;
 
+  /**
+   * The smallest container ID in the current response batch.
+   * Used for pagination to determine the lower bound for the next page.
+   */
+  @JsonProperty("firstKey")
+  private long firstKey = 0;
+
+
+  /**
+   * The largest container ID in the current response batch.
+   * Used for pagination to determine the upper bound for the next page.
+   */
+  @JsonProperty("lastKey")
```

Comment:
These same changes appear in https://github.com/apache/ozone/pull/8796 , may need rebase of code ?